### DISCUSSION TITLE
Cheating

### DISCUSSION THREAD

**Discussion by pillaivishal** [SafeExamBrowser/seb-win-refactoring#397] [Q&A] [answered]
Hello everyone we asked our student to download & use seb but one of our student was able to run seb in virtual machine with few minutes of installing the latest version(he did not change the code or any setting of the seb) he made few changes in virtual machine and went on  completing the test in virtual machine please help me he is in  class 12

**Comment by dbuechel** [comment, marked as answer]
There is nothing we can do about this issue at the moment as VM detection is incredibly difficult and error-prone (and in some cases even impossible, I'd argue). The VM detection mechanism has been improved and extended also by community members, but it is still far from perfect i.e. any person having sufficient knowledge will be able to circumvent it.

**Reply by artemko-ct** [reply]
Is there a way to disable virtual machine detection for debugging purposes on macOS?

**Comment by pillaivishal** [comment]
ok
thank you

**Comment by aurelienb1** [comment]
In BYOD there is a simple way for supervisors to check this, normally in a virtual machine SEB indicates that it does not have access to WIFI and the WiFi network icon is crossed out in SEB. So the student can take the exam without being connected to WiFi which should raise the alarm. There is a way around this, but not all students can do it.
